cmd/commands/function.go

```diff
@@ -184,3 +188,43 @@ func FunctionUpdate(buildpackBuilder core.Builder, fcTool *core.Client) *cobra.C
 
 	return command
 }
+
+func FunctionBuild(buildpackBuilder core.Builder, fcTool *core.Client, defaults PackDefaults) *cobra.Command {
+	buildFunctionOptions := core.BuildFunctionOptions{
+		BuildOptions: core.BuildOptions{
+			BuildpackImage: defaults.BuilderImage,
+			RunImage:       defaults.RunImage,
+		},
+	}
+
+	command := &cobra.Command{
+		Use:   "build",
+		Short: "Build a function container from local source",
```

Comment:
I think it would be helpful to add a long description with examples
it still takes me some time to adjust to `--artifact` and `--handler`,  I guess it could be the case to first-time users as well

Reply:
I'll centralize the flags between create and build, and we can refine the help in a future PR.

Reply:
👍